libraries/tokens/src/types/clickbound.ts

```diff
@@ -1,7 +1,7 @@
 export type Clickbound = {
-  height: string
+  height?: string
   width?: string
-  offset: {
+  offset?: {
```

Comment:
Not sure I can see why these were set to optional?

Reply:
True, I found out I didn't need it at some point but forgot to remove it 🔥